### PR TITLE
Management task for retrying schedule deliveries that have half completed.

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
ready for review but this depends on https://github.com/praekelt/seed-message-sender/pull/19 merging in first.

File: scheduler/management/commands/trigger_periodic_task.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def confirm(prompt):
                 model_entry.args, model_entry.kwargs,
                 **model_entry.options)
         else:
-            app.send_task(
+            async_result = app.send_task(
```
Comment (smn) [inline scheduler/management/commands/trigger_periodic_task.py:76]: This was unrelated to this PR, it's a bug fix from last week's hacking in Amsterdam to do a manual periodic task trigger

File: scheduler/management/commands/trigger_deliver_tasks.py [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+
+class MessageSenderApiClient(object):
+    """
+    Client for Message Sender Service.
+    :param str auth_token:
+        An access token.
+    :param str api_url:
+        The full URL of the API.
+    """
```
Comment (smn) [inline scheduler/management/commands/trigger_deliver_tasks.py]: This should move to the `seed-services-client` package.

Comment (smn) [reply scheduler/management/commands/trigger_deliver_tasks.py]: https://github.com/praekeltfoundation/seed-services-client/pull/15 is the PR for that, when that lands we can remove this code.

**Comment by smn** [comment]
@jpaidoussi ready for review

**Review by jpaidoussi — APPROVED** [review]

**Pull request merged**